### ISSUE TITLE
`filter` key unsupported in `\DeclarePrintbibliographyDefaults`  

### ISSUE THREAD

**Issue by eah13** [plk/biblatex#1262]
The documentation lists `filter` as a supported key for  `\DeclarePrintbibliographyDefaults` , yet this MWA yields an error complaining that the key is unknown:

```latex
\documentclass{article}

\usepackage{biblatex}
\DeclarePrintbibliographyDefaults{filter=anything}

\begin{document}

\nocite{*}
\printbibliography

\end{document}
```

(Bug happens early enough that it doesn't matter that the filter name is invalid, there is no `\addbibresource`, etc.)

I took a look at the [commit adding  `\DeclarePrintbibliographyDefaults` ](https://github.com/plk/biblatex/commit/b22f1578d64436de57858f9c05a3a13c3ca0e49c) and sure enough, looks like [there's no `\define@key{blx@bibdefaults}{filter}`](https://github.com/plk/biblatex/commit/b22f1578d64436de57858f9c05a3a13c3ca0e49c#diff-6c60f2550cf6c831795d89e5f8e914df013a7d66374d44622bb9e872f591f8f3L8957-L9137) or [`\let\blx@theenv\blx@default@thefilter`](https://github.com/plk/biblatex/blame/b22f1578d64436de57858f9c05a3a13c3ca0e49c/tex/latex/biblatex/biblatex.sty#L9113-L9134) (assuming that would be the correct syntax?) alongside the other keys.

It's very possible I have misunderstood the documentation and/or code. If so, apologies, and thanks in advance for any education you can provide.

I hope this is an omission in that commit, not an error to the documentation: setting a default filter would be very useful! (Notwithstanding the fact I seem to be the first person to report this bug...)

Assuming I'm not confused or that this is a documentation error, here's some minimal code that would facilitate testing the implementation of a fix:

<details>
  <summary>MWA for testing</summary>

```latex
\documentclass{article}

% Pick any author name, e.g. "Smith"
\usepackage[style=publist, plauthorname=Smith]{biblatex}

% Use any bib with "Smith" and non-"Smith" authored entries 
\addbibresource{mwa.bib} 

% publist defines the `mine` filter, which is built from the category `filtered`,
% which is all the entries that match the plauthor nameparts.
\DeclarePrintbibliographyDefaults{filter=mine}

\begin{document}

\nocite{*}
\printbibliography

\end{document}
```

**Result: only entries with author lastname "Smith" are printed.**

___

</details>

I don't fully understand the ways that `filter` might need a different implementation than the other keys, so refrained from attempting a PR. Also, Lest my source code spelunking misrepresent me, I  am not a package author (one day, hopefully).

Thanks for a great package and all the hard work that goes into it.

**Comment by plk** [comment]
Fixed in 3.19 DEV

**Comment by plk** [comment]
Fixed in 3.19 release
